debsources/app/copyright/templates/copyright/index.html

```diff
@@ -4,7 +4,8 @@
   https://anonscm.debian.org/gitweb/?p=qa/debsources.git;a=blob;f=AUTHORS;hb=HEAD
   License: GNU Affero General Public License, version 3 or above.
 #}
-{# copied from templates/index.html #}
+{# XXX copied from templates/index.html #}
+{# this should be a macro: repetitive code which makes change awkward #}
```

Comment:
Fixed this, will push up soon to the related branches. I'll review diffs and make sure there's not much more